ztp/siteconfig-generator/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+
+	siteConfigs "github.com/openshift-kni/cnf-features-deploy/ztp/siteconfig-generator/siteConfig"
+	"github.com/spf13/pflag"
```

Comment:
Though honestly for all we're doing with it, don't bother with pflag, and just `import "flag"`, the standard go module: https://pkg.go.dev/flag

I think if you change the import to "flag" and then just globally `s/pflag/flag/` it will work as-is!

Reply:
I checked; it does :)

Reply:
thanks :) , done